src/harness/sas_test_harness.py

```diff
@@ -284,71 +284,74 @@ def generateCbsdReferenceId(fcc_id, serial_number):
   """
   return str(fcc_id + '/' + str(hashlib.sha1(serial_number).hexdigest())).encode('utf-8')
 
-def generateCbsdRecords(cbsd_records, grant_records_list):
-  """ Generates the cbsdData Object by combining cbsdrecord and grantrecord based on SAS-SAS TS.
-
+def generateCbsdRecords(registration_requests, grant_requests_list):
+  """ Generates the cbsdData Object by combining registration request and grant request based
+      on SAS-SAS TS.
   Args:
-     cbsd_records: list of cbsdData objects that includes one or more grants object.
-     grant_records_list: list of muliple grant records for all cbsd devices.
+     registration_requests: list of registration requests that includes one or more grant requests.
```

Comment:
I don't think you mean that the registration request should contain a grant request. Maybe you mean that the corresponding CBSDs will have at least one corresponding Grant request?

Reply:
Description is rephrased